screens/cart/cart.css

```diff
@@ -0,0 +1,42 @@
+@import url(/css/global.css);
+
+
+.btn {
+    width: 100%;
+    padding: 1.5rem 0;
+    margin: 1rem 0;
+}
+
+.cart .card-image {
+    width: 28.8rem;
```

Comment:
giving fix width might lead to responsiveness issue,max-width might be a good option

Reply:
Done